cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -223,6 +260,9 @@ public CloudantClient build() {
         //Build properties and couchdb client
         CouchDbProperties props = new CouchDbProperties(url);
 
+        props.addRequestInterceptors(USER_AGENT_INTERCEPTOR);
```

Comment:
Wouldn't it be better to just pass the same interceptor around? It doesn't mutate anything internally, so I don't think threading is an issue.